schemas/yaml/MultiLangString.yml

```diff
@@ -1,26 +1,28 @@
 title:   'Multi-Language Text / String'
 
-description: 'A piece of text in multiple languages. This is typically used for analyses, database properties, and information in metalanguages rather than transcriptions or translations of data (for those, see the [Transcription](https://spec.digitallinguistics.github.io/schemas/Transcription.html) and [Translation](https://spec.digitallinguistics.github.io/schemas/Translation.html) schemas). For example, the part of speech for a lexeme might be given as both English *noun* and Spanish *nombre*. Each key must be the abbreviation for a language (see the [Abbreviation](https://spec.digitallinguistics.github.io/schemas/Abbreviation.html) schema), and the value for that key is the string of text in that language. **Note:** MultiLangStrings do not have a `type` property.
+description: 'A piece of text in multiple languages. This is typically used for analyses, database properties, and information in metalanguages rather than transcriptions or translations of data (for those, see the [Transcription](https://spec.digitallinguistics.github.io/schemas/Transcription.html) and [Translation](https://spec.digitallinguistics.github.io/schemas/Translation.html) schemas). For example, the part of speech for a lexeme might be given as both English *noun* and Spanish *nombre*. Each key must be the valid IETF language tag for a language (see the [IETF](https://www.w3.org/International/articles/language-tags/) reference page), and the value for that key is the string of text in that language. **Note:** MultiLangStrings do not have a `type` property.
 
-    If the text is in only one language and that language is English, it may be represented as a simple string instead of an object. Thus instead of `gloss: { eng: "M.SG" }` you can simply write `gloss: "M.SG"`.'
+    If the text is in only one language and that language is English, it may be represented as a simple string instead of an object. Thus instead of `gloss: { en: "M.SG" }` you can simply write `gloss: "M.SG"`.'
 
 oneOf:
 
   - type:                 object
     additionalProperties: false
     patternProperties:
-      '^[(a-z)|(A-Z)|(0-9)]+$':
+      '^(((en-GB-oed|i-ami|i-bnn|i-default|i-enochian|i-hak|i-klingon|i-lux|i-mingo|i-navajo|i-pwn|i-tao|i-tay|i-tsu|sgn-BE-FR|sgn-BE-NL|sgn-CH-DE)|(art-lojban|cel-gaulish|no-bok|no-nyn|zh-guoyu|zh-hakka|zh-min|zh-min-nan|zh-xiang))|((([A-Za-z]{2,3}(-([A-Za-z]{3}(-[A-Za-z]{3}){0,2}))?)|[A-Za-z]{4}|[A-Za-z]{5,8})(-([A-Za-z]{4}))?(-([A-Za-z]{2}|[0-9]{3}))?(-([A-Za-z0-9]{5,8}|[0-9][A-Za-z0-9]{3}))*(-([0-9A-WY-Za-wy-z](-[A-Za-z0-9]{2,8})+))*(-(x(-[A-Za-z0-9]{1,8})+))?)|(x(-[A-Za-z0-9]{1,8})+))$':
         title: Text
         type:  string
-        description: 'The string of text in the specified language'
+        description: 'The string of text in the specified language
+
+                      The IETF regex was created with the function found [here](https://www.npmjs.com/package/ietf-language-tag-regex).'
```

Comment:
Let's make this a YAML comment, rather than part of the description itself. You can do this by starting the comment line with a `#`. End users of this format don't really need to know where the regex came from - it's more a note for developers who might be looking at this source code.

Same for the other locations you added this note.